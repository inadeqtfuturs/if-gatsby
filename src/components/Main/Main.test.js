import React from 'react';
import { Main } from '@components';
import ThemeRender from '@utils/testing';

describe('Main', () => {
  it('renders correctly', () => {
    const tree = ThemeRender(
      <Main>
        <p>hello world</p>
      </Main>
    );
    expect(tree).toMatchSnapshot();
  });
});
