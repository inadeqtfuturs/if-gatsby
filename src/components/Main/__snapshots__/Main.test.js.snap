// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders correctly 1`] = `
<main
  className="sc-fzoyAV lgNqAT"
>
  <div
    className="sc-AxirZ gXXnrR"
    height="100%"
    width={
      Array [
        null,
        "540px",
        "720px",
        "960px",
        "1140px",
        "1320px",
      ]
    }
  >
    <p>
      hello world
    </p>
  </div>
</main>
`;
